exercise-1/script.js

```diff
@@ -0,0 +1,22 @@
+const button = document.querySelector('button');
+const nav = document.querySelector('nav');
+const main = document.querySelector('.hide-main');
+
+button.addEventListener('click', function () {
+  nav.classList.toggle('display');
+  main.classList.toggle('hide-main');
+});
+
+function keyPress(e) {
+  if (e.key === 'Escape') {
+    console.log('hello');
+  } else false;
+}
+
+document.onkeydown = function (evt) {
```

Comment:
Two things here:
1. Instead of assigning a value to `document.onkeydown`, it's better to use `addEventListener`. There can be multiple event listeners attached to the same node, and they can also be removed.
2. Abbreviating `event` to `evt` made me cringe a bit. 😬 The convention is using `e` instead. I personally just write `event`, or even go more descriptive and write `keyDownEvent` or `documentKeyDownEvent`. Characters are cheap. Understandability is priceless.